doc/tutorial/argument_parser/index.md

```diff
@@ -411,16 +416,16 @@ Check if your options are set correctly by trying the following call:
 
 # Validation of (positional) option values {#section_validation}
 
-Our applications often do not allow just any value to be passed as input arguments and if we do not check for them, 
-the program may exhibit undefined behaviour. The best way to carefully restrict user input is to directly check the 
+Our applications often do not allow just any value to be passed as input arguments and if we do not check for them,
+the program may exhibit undefined behaviour. The best way to carefully restrict user input is to directly check the
 input when parsing the command line. The seqan3::argument_parser provides **validators** for a given (positional) option.
 
-A *validator* is a [functor](https://stackoverflow.com/questions/356950/what-are-c-functors-and-their-uses) that is 
+A *validator* is a [functor](https://stackoverflow.com/questions/356950/what-are-c-functors-and-their-uses) that is
 called within the argument parser after retrieving and converting a command line argument. We provide several validators,
 which we hope cover most of the use cases, but you can always create your own validator
-(see section [Create your own validator](#section_create_your_own_validator)).
+(see section [Create your own validator](#cookbook_custom_validator)).
```

Comment:
Is this change intentional? Was the linking not correct before?

Reply:
My branch was behind the current master. The `(#cookbook_custom_validator)` version is correct and also in the current  master.